.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1919,12 +1931,12 @@ public class FineTuningClient {
         public virtual Task<ClientResult> CreateJobAsync(BinaryContent content, RequestOptions options = null);
         public virtual ClientResult GetJob(string jobId, RequestOptions options);
         public virtual Task<ClientResult> GetJobAsync(string jobId, RequestOptions options);
-        public virtual IEnumerable<ClientResult> GetJobCheckpoints(string jobId, string after, int? limit, RequestOptions options);
-        public virtual IAsyncEnumerable<ClientResult> GetJobCheckpointsAsync(string jobId, string after, int? limit, RequestOptions options);
-        public virtual IEnumerable<ClientResult> GetJobEvents(string jobId, string after, int? limit, RequestOptions options);
-        public virtual IAsyncEnumerable<ClientResult> GetJobEventsAsync(string jobId, string after, int? limit, RequestOptions options);
-        public virtual IEnumerable<ClientResult> GetJobs(string after, int? limit, RequestOptions options);
-        public virtual IAsyncEnumerable<ClientResult> GetJobsAsync(string after, int? limit, RequestOptions options);
+        public virtual CollectionResult GetJobCheckpoints(string jobId, string after, int? limit, RequestOptions options);
```

Comment:
FineTuningClient is missing the new constructors that take a `string apiKey` parameter.